src/sentry/shared_integrations/client.py

```diff
@@ -62,14 +62,14 @@ def from_response(self, response, allow_text=False):
 
         # Some APIs will return JSON with an invalid content-type, so we try
         # to decode it anyways
-        if "application/json" not in response.headers["Content-Type"]:
+        if "application/json" not in response.headers.get("Content-Type", {}):
```

Comment:
The default here shouldn’t be a dict, it’s a string.

Reply:
@mattrobenolt Good catch! Fixed.